### DISCUSSION TITLE
After 2.0.1 Noice opens up the message window upon startup :: annoying

### DISCUSSION THREAD

**Discussion by sinkr** [folke/noice.nvim#812] [Q&A]
...and it's annoying.  If I roll the repo to `09102ca2e9a3e9302119fdaf7a059a034e4a626d` it stops.  I guess I'll either pin the version here or hope that it gets fixed because the 1st thing I do when I go to edit a file is have all my keystrokes captured by the fsck-n Noice window.
![noice](https://github.com/folke/noice.nvim/assets/696374/0c592784-db07-4ebf-b521-03c71c3fb512)

**Comment by folke** [comment]
I just changed it. Will no longer popup, but still show in history

**Comment by sinkr** [comment]
Thank you very much!
